portal/code/cx-portal-shared-components/src/components/basic/Table/components/RoleChip.tsx

```diff
@@ -0,0 +1,11 @@
+import './RoleChip.scss'
+// Temporary component to visualise different table cell
+const RoleChip = ({
+                         role
+                       }: any) => {
```

Comment:
I suggest we all run a `yarn pretty` or before checking in the code so we have a uniform code style

Reply:
done